src/Workspaces/VisualBasicTest/CodeGeneration/SyntaxGeneratorTests.vb

```diff
@@ -2786,6 +2786,77 @@ End Interface</x>.Value)
         End Sub
 
         <Fact>
+        <WorkItem(5097, "https://github.com/dotnet/roslyn/issues/5097")>
+        Public Sub TestAddInterfaceWithEOLs()
+            Dim classC = SyntaxFactory.ParseCompilationUnit("
+Public Class C
+End Class").Members(0)
+
+            VerifySyntaxRaw(Of ClassBlockSyntax)(
+                _g.AddInterfaceType(classC, _g.IdentifierName("X")), "
+Public Class C
+ImplementsXEnd Class")
```

Comment:
These tests a asserting the preformatted state after the operation is applied.  None of the SyntaxGenerator API add/manipulate trivia. All scenarios using SyntaxGenerator eventually hit the formatter.